disk/smart-monitoring.sh

```diff
@@ -154,7 +155,7 @@ then
     # Write conf to file
     # https://wiki.debianforum.de/Festplattendiagnostik-_und_%C3%9Cberwachung#Beispiel_3
     echo "DEVICESCAN -a -I 194 -W 5,45,55 -r 5 -R 5 -n standby,24 -m <nomailer> -M exec \
-$SCRIPTS/smart-notification.sh -s (O/../../(2|4|6)/02|S/../../5/02|L/../20/./02)" > /etc/smartd.conf
+$SCRIPTS/smart-notification.sh -s (S/../.././01|L/../../6/02)" > /etc/smartd.conf
```

Comment:
This change refers to https://github.com/nextcloud/nextcloudpi/blob/be30663c7a4b86a4b91bf39b3b2d49c87f53d95c/bin/ncp/SYSTEM/nc-hdd-monitor.sh#L68